modules/overview-of-kueue-resources.adoc

```diff
@@ -0,0 +1,136 @@
+:_module-type: CONCEPT
+
+[id='overview-of-kueue-resources_{context}']
+= Overview of Kueue resources
+
+[role='_abstract']
+The cluster administrator can configure Kueue resource flavors, cluster queues, and local queues to manage distributed workload resources across multiple nodes in an OpenShift cluster.
```

Comment:
If this content is intended for multiple personas, would it not be better to write "Cluster administrators can configure....".  There could be multiple cluster administrators with the cluster-admin permission.